README.md

```diff
@@ -13,7 +13,27 @@ But the resources can be varied based on availability. However, make sure to
 use `--iso-url centos` part in above command as we have setup things on CentOS
 based minishift VM.
 
-Once the VM is ready, spin up a Jenkins server that can be used by the Jenkins
+Once the VM is ready, get into the vm using the command `$ minishift ssh`. Once you are in, apply the following steps to install an external registry. You are free to install it on a separate system, or your base machine, just ensure it is reachable from the minishift vm.
+
+    $ yum -y install docker-distribution
+
+If you are running on a separate machine, as it would be recommended, you can skip the below step of configuring the registry. Otherwise please edit the config file located at `/etc/docker-distribution/registry/config.yml` to setup ports/storage etc. Once done, start the service with
+
```

Comment:
Please share an example setting of ports, storage, etc. as you've used. Also, are you sure it's `ports/storage etc.` and not `ports, storage, etc.`?